src/cmd-buildupload

```diff
@@ -75,6 +75,11 @@ def cmd_upload_s3(args):
         # assume it's metadata
         s3_copy(f'builds/{args.build}/{f}', bucket, f'{prefix}/{args.build}/{f}',
                 CACHE_MAX_AGE_METADATA, args.acl)
+
+    # Copy build logs to S3
+    s3_copy('job.log', bucket, f'{prefix}/{args.build}/build.log',
```

Comment:
I think you should test for the existence of `job.log` before trying to copy it.

We know that RHCOS pipelines have it present, but it's unclear if the FCOS pipelines do too.

Reply:
From looking at the FCOS pipeline, it doesn't seem like they do. I'll add the check for whether the file exists.

Reply:
Rather than special-casing this in `cosa buildupload`, would it make more sense to just copy the file in the build dir instead in the pipeline? I've thought about something similar for FCOS as well.

Then it'll automatically get uploaded as part of the upload. (This is the kind of thing the "e.g. pipelines might inject additional metadata" comment above refers to. :) ).

Fine with this too!

Reply:
That's a good point. I agree that making a change in the RHCOS pipeline is a cleaner solution than the special-casing here. I'll go with that change instead. Thanks for the input!